### PR TITLE
CS improvements

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Ran

vendor/bin/phpcbf src --standard=phpcs.xml --tab-width=4
vendor/bin/phpcbf tests --standard=phpcs.xml --tab-width=4

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -8,10 +8,10 @@
 	die( 'You need to install this package with Composer before you can run the tests' );
 }
 
-$classLoader = require_once( __DIR__ . '/../vendor/autoload.php' );
+$classLoader = require_once ( __DIR__ . '/../vendor/autoload.php' );
```
Comment (thiemowmde) [inline tests/bootstrap.php]: Uh?

Comment (JeroenDeDauw) [inline tests/bootstrap.php]: wat.png

Comment (thiemowmde) [inline tests/bootstrap.php]: The added space is very odd. This should either be `function( $string )` or `keyword $string`.

Comment (JeroenDeDauw) [inline tests/bootstrap.php]: #66

**Pull request merged**